<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkToFConnectionWidgetControls2</class>
 <widget class="QWidget" name="QmitkToFConnectionWidgetControls2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>229</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkToFConnection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QmitkToFPMDParameterWidget" name="m_PMDParameterWidget" native="true"/>
   </item>
   <item row="5" column="0">
    <widget class="QmitkStructureSensorParameterWidget" name="m_StructureParameterWidget" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="QmitkToFMESAParameterWidget" name="m_MESAParameterWidget" native="true"/>
   </item>
   <item row="4" column="0">
    <widget class="QmitkKinectParameterWidget" name="m_KinectParameterWidget" native="true"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QmitkServiceListWidget" name="m_DeviceList" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_ConnectCameraButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Connect to camera</string>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/QmitkToFUtilWidget.qrc">
         <normaloff>:/images/powerRed.png</normaloff>
         <normalon>:/images/powerGreen.png</normalon>:/images/powerRed.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ToF camera connection</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkToFPMDParameterWidget</class>
   <extends>QWidget</extends>
   <header>QmitkToFPMDParameterWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkToFMESAParameterWidget</class>
   <extends>QWidget</extends>
   <header>QmitkToFMESAParameterWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkKinectParameterWidget</class>
   <extends>QWidget</extends>
   <header>QmitkKinectParameterWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkServiceListWidget</class>
   <extends>QWidget</extends>
   <header>QmitkServiceListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkStructureSensorParameterWidget</class>
   <extends>QWidget</extends>
   <header>QmitkStructureSensorParameterWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/QmitkToFUtilWidget.qrc"/>
 </resources>
 <connections/>
</ui>
